src/averageAges.js

```diff
@@ -54,8 +72,29 @@ function calculateWomenAverageAge(people, withChildren) {
  *
  * @return {number}
  */
+function findMother(people, child) {
+  return people.some(mother => mother.name === child.mother);
+}
+
+function motherBorn(people, mother) {
+  const mom = people.find(person => person.name === mother);
+
+  return mom.born;
+}
+
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const child = people.filter(person => (
```

Comment:
name children for this constant would be more appropriate because the .filter() method will return an array of objects  — children instead of single one child.

Reply:
agree. If you receive an array, you can't name it 'child', as it's not one child. It's children